torch/_inductor/graph.py

```diff
@@ -590,8 +597,8 @@ def compile_to_module(self):
         for name, value in self.constants.items():
             setattr(mod, name, value)
 
-        if dynamo_config.output_code:
-            log.info("Output code: %s", mod.__file__)
+        if config.output_compiled_module_path:
```

Comment:
I'm also a little bit uncertain about what's changing here and why we need 2 environmental variables for this?

Reply:
So if we enable output_code config other things like
- guards
- fx graph
are also printed.

Added another environment variable so when enabling it, we just specifically print the compiled module path.

Reply:
I guess it seems weird to me to add 2 new env variables for one piece of new functionality.  In particular, this change seems weird since we're just changing some logging info for the pre-existing config option.

Reply:
I see. Would it be better if I only add config.benchmark_kernel . Print the compiled module path if that config is true?

BTW, the reason I change log.info to print is because enable info log just cause too much other stuff being being printed. So I print directly to stderr.